config/initializers/statsd.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "datadog/statsd"
+
+class StubStatsd
+  def increment; end
```

Comment:
Could we make this match the [original definition](https://github.com/DataDog/dogstatsd-ruby/blob/master/lib/datadog/statsd.rb#L120-L131)?

```ruby
class StubStatsd
  # https://github.com/DataDog/dogstatsd-ruby/blob/d77b4bbaa734b2e7b02875c588fccc4221864276/lib/datadog/statsd.rb#L120-L131
  def increment(stat, opts={}); end
end
```